script/Arcueid/Day 9/09_00_ARC09_1.txt

```diff
@@ -26,6 +26,7 @@ Things that my father ordered me to forget.
 [sha:2ce4023198d762d748648342d653f8bf32994ce0]{
 -- Page 2, Offset 21297.
 -- 夏の、暑い日。
+//A hot summer day is used more commonly in prose
```

Comment:
ngrams seems to agree. The linter is mad because this line is used elsewhere, so just be sure to update 11_01_CIEL11_1 to match.